app/code/local/Magmodules/Webwinkelconnect/etc/system.xml

```diff
@@ -198,6 +198,28 @@
                         </resend_double>
                     </fields>
                 </invitation>
+                <product_review_invites>
+                    <label>Invitation to write product review</label>
+                    <frontend_type>text</frontend_type>
+                    <sort_order>3</sort_order>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>1</show_in_website>
+                    <show_in_store>1</show_in_store>
+                    <expanded>1</expanded>
+                    <comment><![CDATA[WebwinkelKeur can invite your customers to write product reviews.<br/><br/>]]></comment>
+                    <fields>
+                        <enabled>
+                            <label>Activate</label>
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <sort_order>1</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>1</show_in_store>
+                            <comment>Send product review invitation?</comment>
```

Comment:
It's interesting that there are no `translate` attributes for the comments. Are those always in English in the admin interface, or how does that work?

Reply:
<product_review_invites translate="label,comment">
this translates the label and the comment as long as the translations are added to the translation .csv file.